apps/file/serializers.py

```diff
@@ -1,15 +1,50 @@
 from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
 
 from apps.file.models import File
+from utils.common import get_file_extension
+
+MAX_FILE_SIZE_MB = 10  # 10 MB limit
+MAX_FILE_SIZE_BYTES = MAX_FILE_SIZE_MB * 1024 * 1024
```

Comment:
It's better to move this const to `settings.py`

Reply:
moved it to `consts.py` instead of `settings.py`. I think it's better to keep django specific constants in `settings.py` and store other constants in a different place. Otherwise I fear `settings.py` will be too crowded.